frontend/src/components/App/PluginSettings/PluginSettings.tsx

```diff
@@ -284,15 +278,69 @@ export function PluginSettingsPure(props: PluginSettingsPureProps) {
 export default function PluginSettings() {
   const dispatch = useDispatch();
 
-  const pluginSettings = useTypedSelector(state => state.plugins.pluginSettings);
+  const pluginData = useTypedSelector(state => state.plugins.pluginData);
+
+  const oldLocalEnabledList = localStorage.getItem('headlampPluginSettings');
+  const localEnabledList = localStorage.getItem('enabledPluginsList');
+
+  let pluginsEnabledList;
+
+  /**
+   * Need to improve later, this is a temporary fix to handle the migration of the old plugin settings.
```

Comment:
what exactly needs to be improved here? please mark it as todo and add information on when this needs to be done (if you can)

Reply:
this is somewhat of a meta comment from a meeting I had with joaquim where essentially we should having some backwards compatibility for now but eventually phasing it out to just use the new way

Reply:
Removing wording of 'to do' tone and making it a note